README.rst

```diff
@@ -13,45 +13,36 @@ Beamline Simulation using `sirepo-bluesky`_ and Synchrotron Radiation Workshop (
 
 Purpose
 -------
-
-Use machine learning to simulate beamlines similar to SRW with lower computational costs.
-The goal is to determine whether or not a neural network can construct a beamline simulation given data from sirepo-bluesky.
+Use machine learning techniques to simulate beamlines given data from SRW.
 
 Installation
 ------------
 
 - Install `VirtualBox`_ on your computer.
 - Install `Vagrant`_ using the terminal.
 - Git clone this repository. Then use ``cd deep-beamline-simulation`` to move into the top level directory of this repository.
-- You will see a Vagrantfile containing virtual machine setup information. Start the virtual machine using ``vagrant up`` followed by ``vagrant ssh``.
-- Once the virtual machine is running, finish the Miniconda installation,
-.. code:: bash
+- You will see a Vagrantfile containing virtual machine setup information. Start the virtual machine using ``vagrant up`` followed by ``vagrant ssh``. To reload the virtual machine use ``vagrant reload`` and to update the machine for any changes use ``vagrant provision``.
 
-   bash Miniconda3-latest-Linux-x86_64.sh
-
-- Close and reload the VM. Create a conda environment, 
-.. code:: bash
+- It is recommended to check the status of Mongo DB using ``sudo systemctl status mongod``. If the status is 'dead' use ``sudo systemctl start mongod.service`` to start running Mongo DB.
```

Comment:
Also you can leave off ``.service`` from ``mongod.service`` in ``sudo systemctl start mongod.service``.